Tools/scripts/docker_build.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
One of the few places I've ever seen `/usr/bin/env bash` :-)

Reply:
When you work with Nix for a bit, you pick this up :P